.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,10 +11,10 @@ assignees: mrjesen
 A clear and concise description of what the bug is.
 
 **Client Software**
-HMCL (MC: 1.16.1)
+HMCL (MC: 1.16.5)
 
 **mods**
- - optfine H3 G2
+ - OptiFine
```

Comment:
I think this line is wrong.You should know this is a bug_report template not a description file,so the more detailed the more the devloper will know.

Reply:
I updated the optifine text because the "i" was missing from it's text string, but the version part can be restored if capable.